docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -4,15 +4,17 @@ id: kubernetes_tutorial
 layout: subpages
 description: Secretless Broker Documentation
 permalink: /docs/get_started/kubernetes_tutorial.html
+redirect_to: /tutorials/kubernetes/kubernetes-tutorial-base.html
+
 ---
 
-This is a detailed, step-by-step tutorial. 
```

Comment:
do we still need the content here if we're redirecting? probably best to delete it, so that people don't get confused if they have to edit this tutorial

Reply:
good catch